pkgs/development/libraries/libtickit/default.nix

```diff
@@ -1,54 +1,67 @@
 { lib
 , stdenv
 , fetchFromGitHub
+, nix-update-script
 , pkg-config
 , libtool
 , perl
 , libtermkey
 , unibilium
 }:
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "libtickit";
   version = "0.4.3";
 
   src = fetchFromGitHub {
     owner = "leonerd";
     repo = "libtickit";
-    rev = "v${version}";
+    rev = "v${finalAttrs.version}";
     hash = "sha256-QCrym8g5J1qwsFpU/PB8zZIWdM3YzOySknISSbQE4Sc=";
   };
 
+  enableParallelBuilding = true;
+
+  #
+  # Both the build and the check phase need a libtool path
+  #
   makeFlags = [
-    "PREFIX=$(out)"
     "LIBTOOL=${lib.getExe libtool}"
   ];
 
+  installFlags = [
+    "PREFIX=${placeholder "out"}"
+  ];
+
   nativeBuildInputs = [
     pkg-config
     libtool
   ];
+
   buildInputs = [
     libtermkey
     unibilium
   ];
+
   nativeCheckInputs = [ perl ];
 
   patches = [
-    ./skipTestMacOS.patch
+    ./001-skip-test-18term-builder-on-macos.patch
   ];
 
   doCheck = true;
 
-  meta = with lib; {
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
     description = "A terminal interface construction kit";
     longDescription = ''
       This library provides an abstracted mechanism for building interactive full-screen terminal
       programs. It provides a full set of output drawing functions, and handles keyboard and mouse
       input events.
     '';
     homepage = "https://www.leonerd.org.uk/code/libtickit/";
-    license = licenses.mit;
-    maintainers = with maintainers; [ onemoresuza ];
-    platforms = platforms.unix;
+    license = lib.licenses.mit;
```

Comment:
This does not really matter and is not worth changing

Reply:
Went back to the `with lib;` statement.